boards/grand_central_m4/examples/usb_serial.rs

```diff
@@ -85,66 +81,66 @@ fn main() -> ! {
     }
 
     unsafe {
-        core.NVIC.set_priority(interrupt::USB_OTHER, 1);
         core.NVIC.set_priority(interrupt::USB_TRCPT0, 1);
-        core.NVIC.set_priority(interrupt::USB_TRCPT1, 1);
-        NVIC::unmask(interrupt::USB_OTHER);
         NVIC::unmask(interrupt::USB_TRCPT0);
+        core.NVIC.set_priority(interrupt::USB_TRCPT1, 1);
         NVIC::unmask(interrupt::USB_TRCPT1);
+        core.NVIC.set_priority(interrupt::USB_SOF_HSOF, 1);
+        NVIC::unmask(interrupt::USB_SOF_HSOF);
+        core.NVIC.set_priority(interrupt::USB_OTHER, 1);
+        NVIC::unmask(interrupt::USB_OTHER);
     }
 
+    // Flash the LED in a spin loop to demonstrate that USB is
+    // entirely interrupt driven.
     loop {
-        let pending = disable_interrupts(|_| unsafe {
-            let pending = PENDING_COLOR;
-            PENDING_COLOR = None;
-            pending
+        cycle_delay(5 * 1024 * 1024);
+        red_led.toggle().unwrap();
+        // Turn off interrupts so we don't fight with the interrupt
+        cortex_m::interrupt::free(|_| unsafe {
+            if USB_BUS.as_mut().is_some() {
+                if let Some(serial) = USB_SERIAL.as_mut() {
+                    let _ = serial.write("Hello USB\n".as_bytes());
+                }
+            }
         });
-        if let Some(color) = pending {
-            let _ = neopixel.write((0..5).map(|_| color));
-        }
     }
 }
 
-static mut USB_ALLOCATOR: Option<UsbBusAllocator<UsbBus>> = None;
-static mut USB_BUS: Option<UsbDevice<UsbBus>> = None;
-static mut USB_SERIAL: Option<SerialPort<UsbBus>> = None;
-static mut PENDING_COLOR: Option<RGB8> = None;
+static mut USB_ALLOCATOR: Option<UsbBusAllocator<UsbBus<Gclk1Id>>> = None;
+static mut USB_BUS: Option<UsbDevice<UsbBus<Gclk1Id>>> = None;
+static mut USB_SERIAL: Option<SerialPort<UsbBus<Gclk1Id>>> = None;
```

Comment:
I wonder if a LazyCell could be used to improve the look of this part. This is not something that I would recommend for this change, just something to think about.